### ISSUE TITLE
Could not find method getNonProxyHosts() after upgrade to 5.3.1

### ISSUE THREAD

**Issue by yellowtrolley** [dependency-check/DependencyCheck#2539]
Upgrading to 5.3.1 dependencyCheckAggregate gradle task fails with:
```
Execution failed for task ':dependencyCheckAggregate'.
error	18-Mar-2020 14:11:06	> Could not find method getNonProxyHosts() for arguments [org.gradle.internal.resource.transport.http.JavaSystemPropertiesSecureHttpProxySettings@616ca765] on task ':dependencyCheckAggregate' of type org.owasp.dependencycheck.gradle.tasks.Aggregate.
```

It works fine with 5.3.0

**Version of dependency-check used**
5.3.1 gradle plugin

**Log**
https://gist.github.com/yellowtrolley/64fe64377af3f8acf0751989cb94fe42

**Comment by Marx314** [comment]
duplicate of this issue : https://github.com/jeremylong/dependency-check-gradle/issues/181

**Comment by jeremylong** [comment]
This will be included in 5.3.2.
